### PR TITLE
Section redevelopment - Application Use/Usage

### PR DESCRIPTION
**Pull request by djw-m**
## What Changed?

Split out, some structure added to lead content pulled differences handling into one doc. Some sections are anaemic and will need expansion. Also, TAMS seems out of place. Differences in version content may be more suited for the expanded upgrades section.

### CODE REVIEW

**Review by LenzGr — DISMISSED** [review]
Comparing the [current](https://www.enterprisedb.com/docs/pgd/latest/appusage/) with the [proposed structure](https://deploy-preview-4873--edb-docs-staging.netlify.app/docs/pgd/latest/appusage/) - LGTM!

File: product_docs/docs/pgd/5/appusage/index.mdx [hunk 1]
```diff
@@ -16,7 +15,7 @@ Developing an application with PGD is predominantly the same as working with any
 
 * [Application behavior](behavior) looks at how PGD replication appears to an application. Which commands are replicated, which run locally, when row-level locks are acquired, how and where triggers fire, large objects, toast, and more are covered.
 
-* [DML and DDL](dml-and-ddl) shows what the differences between the two classes of SQL statement are and how PGD handles them.
+* [DML and DDL replication](dml-and-ddl) shows what the differences between the two classes of SQL statement are and how PGD handles replicating them. It also looks at which commands PGD doesn't replicate at all.
```
Comment (LenzGr) [inline product_docs/docs/pgd/5/appusage/index.mdx:18]:
```suggestion
* [DML and DDL replication](dml-and-ddl) shows what the differences between the two classes of SQL statements are and how PGD handles replicating them. It also looks at which commands PGD doesn't replicate at all.
```

File: product_docs/docs/pgd/5/appusage/dml-ddl.mdx [hunk 2]
```diff
@@ -1,15 +1,27 @@
 ---
-title: DML and DDL replication
-navTitle: DML and DDL
+title: DML and DDL replication amd nonreplication.
```
Comment (LenzGr) [inline product_docs/docs/pgd/5/appusage/dml-ddl.mdx:2]:
```suggestion
title: DML and DDL replication and nonreplication.
```

File: product_docs/docs/pgd/5/appusage/index.mdx [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+* [DML and DDL replication](dml-and-ddl) shows what the differences between the two classes of SQL statement are and how PGD handles replicating them. It also looks at which commands PGD doesn't replicate at all.
+
+* [Nodes with differences](differences) examines how PGD worls with configurations where there are differing table structures and schemas on replicated nodes. Also covered is how to compare between such nodes with LiveCompare and how differences of PostgreSQL versions running on nodes can be handled.
+
+* [Application rules](rules) offers some general rules for applications to avoid data anomalies.
+
+* [Timing considerations](timing) shows how the asynchronous/synchronous replication may affect an applications view of data and notes functions to mitigate stale reads.
+
+* [Table access methods](table-access-methods)(TAMs) notes which TAMs are available with PGD and how to enable them.
```
Comment (LenzGr) [inline product_docs/docs/pgd/5/appusage/index.mdx:26]:
```suggestion
* [Table access methods](table-access-methods) (TAMs) notes which TAMs are available with PGD and how to enable them.
```

File: product_docs/docs/pgd/5/appusage/behavior.mdx [hunk 4]
```diff
@@ -0,0 +1,140 @@
+---
+title: Application behavior
+navTitle: Application behavior
+---
+
+Much of PGD's replication behavior is transparent to applications. Understanding how it
+achieves that and what elements are not transparent is important to successfully Developing
```
Comment (LenzGr) [inline product_docs/docs/pgd/5/appusage/behavior.mdx:7]:
```suggestion
achieves that and what elements are not transparent is important to successfully developing
```

**Review by LenzGr — DISMISSED** [review]
LGTM, I left some minor improvement suggestions.

**Review by LenzGr — APPROVED** [review]
LGTM!

**Review by LenzGr — DISMISSED** [review]
LGTM - great improvement. Thank you!

File: product_docs/docs/pgd/5/appusage/nodes-with-differences.mdx [hunk 5]
```diff
@@ -5,8 +5,8 @@ navTitle: Nodes with differences
 
 ## Replicating between nodes with differences
 
-By default, DDL is automatically sent to all nodes. You can control this
-manually, as described in [DDL replication](ddl), and you can use it to create
+By default, DDL is sent to all nodes. You can control this behavior, 
+as described in [DDL replication](ddl), and you can use it to create
```
Comment (djw-m) [inline product_docs/docs/pgd/5/appusage/nodes-with-differences.mdx:9]:
```suggestion
as described in [DDL replication](../ddl), and you can use it to create
```

**Review by djw-m — COMMENTED** [review]
LGTM (with some link corrections)

**Review by LenzGr — APPROVED** [review]
LGTM 👍

**Pull request merged**